### DISCUSSION TITLE
Setting up the real life timezone of the country

### DISCUSSION THREAD

**Discussion by drfleetles** [EssentialsX/Essentials#4286] [Q&A] [answered]
There is a certain placeholder called {TIME} for the MOTD when you join the server
![image](https://user-images.githubusercontent.com/77469527/123749006-4b322a80-d8f8-11eb-8e66-735aaeb13502.png)
The time is not specific to my time zone and I want to change it. 

Is there any, way I can adjust the clock or set the timezone?

**Comment by pop4959** [comment, marked as answer]
It's the time zone of the server. You can change the time zone through your java startup flags as well.
